code/modules/shuttle/mini_dropship.dm

```diff
@@ -173,6 +173,10 @@
 	open_prompt = FALSE
 	clean_ui_user()
 
+/obj/machinery/computer/camera_advanced/shuttle_docker/minidropship/attack_hand(mob/user)
+	if(damaged == TRUE)
```

Comment:
```suggestion
	if(damaged)
```

Reply:
oh a friend warned me about this but i forgot to chang, thanks